resources/windows/wix/project.wxs.erb

```diff
@@ -20,6 +20,10 @@
       <!--  We are discussing a MAINT PR to move the componentfilegroup to seperate file
             This would allow project to specify additional fragments if needed -->
 
+    <UI>
+      <UIRef Id="WixUI_MainInstallDialog"/>
```

Comment:
I really think this should not be in vanagon hardcoded. WiX allows any of the provided UI dialog sets, a custom one provided by the user, or none specified (in which case it uses the Minimal set). If we hardcode this here with this name, any user **has** to create a custom WiX dialog set **and** name it this, **even** if they don't want a custom dialog set.

I think this can be better expressed by:

```
<UIRef Id="<% settings[:custom_dialog] %>"/> unless setting[:custom_dialog].nil?
```
or something else similar.